src/applications/mhv-landing-page/tests/e2e/medical-records/tool-not-ready-card.cypress.spec.js

```diff
@@ -15,8 +15,8 @@ describe(`${appName} -- transitional Medical records card **enabled**`, () => {
   // eslint-disable-next-line @department-of-veterans-affairs/axe-check-required
   it('renders', () => {
     cy.findByRole('heading', { level: 2, name: heading });
-    cy.findByText(/^The new version of this tool isn’t ready yet./);
-    cy.findByRole('link', { name: /^Go back to the previous version/ });
+    cy.findAllByText(/^The new version of this tool isn’t ready yet./);
+    cy.findAllByRole('link', { name: /^Go back to the previous version/ });
```

Comment:
Question, is there more than one link and more than one instance of this content? I'm not sure I understand why we're using findAllByText

Reply:
And findAllByRole

Reply:
If so, we will need to use the cy.within(selector) syntax to ensure the changes appear within the right HTML element/component 

https://docs.cypress.io/api/commands/within

Reply:
I reverted the change for `cy.findAllByText(/^The new version of this tool isn’t ready yet./);` as that wasn't needed. I used `.findAllByRole('link', { name: /^Go back to the previous version/ });` because that exact text and link appears in the copy change underneath the H1.